tutorials/plot_background_morph.py

```diff
@@ -0,0 +1,875 @@
+potentially scale each value of :math:`y` independent of the respective other.
+
+Now every data point can be transformed independent of other data
+points. This becomes especially handy when morphing volumetric brain data. To
+achieve a mapping of structurally similar areas between subjects, it is
+inevitable to employ non-linear operations to account for morphological
+differences that cannot be represented by a linear transform.
+
+In MNE-Python "brain space" data is represented as source estimate data,
+obtained by one of the implemented source reconstruction methods. See
+:ref:`sphx_glr_auto_tutorials_plot_mne_dspm_source_localization.py`
+
+It is thus represented as :class:`mne.SourceEstimate`,
+:class:`mne.VectorSourceEstimate`, :class:`mne.VolSourceEstimate` or a mixture
+of those.
+
+The data in the first two cases is represented as "surfaces". This means that
+the data is represented as vertices on an inflated brain surface
+representation. In the last case the data is represented in a 4-dimensional
+space, were the last dimension refers to the data's sample points.
+
+Computing an inflated surface representation can be accomplished using
+FreeSurfer. Thereby, spherical morphing of the surfaces can be employed to
+bring data from different subjects into a common anatomical frame. When dealing
+with volumetric data, a non-linear morph map - or better morph volume - based
+on two different anatomical reference MRIs can be computed.
+
+.. _tut_surface_morphing:
+
+Surface morphing
+================
+
+The spherical morphing of the surfaces accomplished by FreeSurfer can be
+employed to bring data from different subjects into a common anatomical
+frame. This chapter describes utilities which make use of the spherical
+morphing procedure. mne_morph_labels morphs label files between subjects
+allowing the definition of labels in a one brain and transforming them to
+anatomically analogous labels in another. mne_average_estimates offers
+the capability to compute averages of data computed with the MNE software
+across subjects.
+
+.. _tut_surface_morphing_maps:
+
+Morph maps
+----------
+
+The MNE software accomplishes surface morphing with help of morphing
+maps which can be either computed on demand or precomputed using
+mne_make_morph_maps , see :ref:`tut_surface_morphing_precompute`. The morphing
+is performed with help of the registered spherical surfaces (``lh.sphere.reg``
+and ``rh.sphere.reg`` ) which must be produced in FreeSurfer .
+A morphing map is a linear mapping from cortical surface values
+in subject A (:math:`x^{(A)}`) to those in another
+subject B (:math:`x^{(B)}`)
+
+.. math::    x^{(B)} = M^{(AB)} x^{(A)}\ ,
+
+where :math:`M^{(AB)}` is a sparse matrix
+with at most three nonzero elements on each row. These elements
+are determined as follows. First, using the aligned spherical surfaces,
+for each vertex :math:`x_j^{(B)}`, find the triangle :math:`T_j^{(A)}` on the
```

Comment:
abuse of notation? if you index `x^{(B)}` from above, you will get a single coordinate. But that is not what you mean, right?